test/unit/job_invocation_report_template_test.rb

```diff
@@ -0,0 +1,78 @@
+require 'test_plugin_helper'
+
+class JobReportTemplateTest < ActiveSupport::TestCase
+  class FakeTask < OpenStruct
+    class Jail < Safemode::Jail
+      allow :action_output, :ended_at
+    end
+  end
+
+  context 'job invocation report template' do
+
+    describe 'correct template is available' do
+      it 'settings select includes only report templates with name job_id' do
+        FactoryBot.create(:report_template, name: 'Template 1')
+        correct_template = FactoryBot.create(:report_template, name: 'Job Invocation Report Template')
+        FactoryBot.create(:template_input, template: correct_template, input_type: 'user', name: 'job_id')
+        templates = Setting::RemoteExecution.job_invocation_report_templates_select
+
+        templates.must_include 'Job Invocation Report Template'
+      end
+
+      it 'should found correct template from setting' do
```

Comment:
This test would be better of and made bit more sense in a `RemoteExecutionHelperTest`. We would test a `job_report_template` method. It would made bit more sense to me :thinking: